src/components/CalendarComponents/categoriesBar.vue

```diff
@@ -37,32 +38,39 @@ import { AllCategories } from '@/db-service/Categories/queries';
   },
 })
 export default class CategoriesBar extends Vue {
-  get categories(): Category[] {
-    if (this.$data.allCategories !== undefined) {
-      return this.$data.allCategories.nodes;
+    @Prop()
+    title!: string;
```

Comment:
שם משמעותי ויש אופציה אולי לנהל את שה כstate פה ולא באבא

Reply:
The categories bar is a dummy component - it's job is to get data and display it.